content/content-strategy/audit-content/prove-value/toc.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: nav/sections
+sections:
```

Comment:
I'm confused; is this article outline supposed to be based on the 'Plan your audit' topic? They're almost identical but too close to have both it seems. Here's the 'Plan your audit' one for comparison @trisha-t 
`+---
+layout: nav/sections
+sections:
+  - Talk about team benefits
+  - Align your goals with agency priorities
+  - Show the costs and efficiencies
+  - Baseline your site
+  - Build empathy with feedback
`

Reply:
trisha fixed